### DISCUSSION TITLE
Highlight country when selected from custom list

### DISCUSSION THREAD

**Discussion by zoom6453** [mullvad/mullvadvpn-app#9018] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

When you select a country inside a custom list, that country is highlighted green inside the list, along with 'All locations' as usual.

And when you click switch location, it'll take you to the currently active country inside whatever list you selected it from. Currently it takes you to the selected country in 'All locations'.
This'll mean you won't have to scroll all the way up if you want to quickly change to a different country inside a list. It'll take you straight to the country inside your list, with the tree expanded displaying your selected country.

### Alternative solutions

Nothing I can do.

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [ ] Android
- [ ] iOS
- [ ] Windows
- [ ] macOS
- [X] Linux

**Comment by raksooo** [comment]
Thank you for suggesting this. It's in our backlog.
